src/Symfony/Component/EventDispatcher/DependencyInjection/EventListenerAutoconfigurator.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\EventDispatcher\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\EventDispatcher\Attribute\EventListener;
+
+/**
+ * @author Alexander M. Turek <me@derrabus.de>
+ */
+final class EventListenerAutoconfigurator
```

Comment:
an inline closure would be easier to read and maintain to me

Reply:
I thought a sharable piece of boilerplate would be nice, but I don't really have strong opinions here. We can inline that.

Reply:
I don't see the need to have this share...